README.md

```diff
@@ -1,5 +1,137 @@
 [![Build Status](https://travis-ci.org/kata-containers/runtime.svg?branch=master)](https://travis-ci.org/kata-containers/runtime)
+[![Build Status](http://kata-jenkins-ci.westus2.cloudapp.azure.com/job/kata-containers-runtime-ubuntu-16-04-master/badge/icon)](http://kata-jenkins-ci.westus2.cloudapp.azure.com/job/kata-containers-runtime-ubuntu-16-04-master/)
+[![Go Report Card](https://goreportcard.com/badge/github.com/kata-containers/runtime)](https://goreportcard.com/report/github.com/kata-containers/runtime)
 
 # Runtime
 
-This project contains the runtime for the Kata Containers project.
+This project contains the runtime for the [Kata Containers](https://github.com/kata-containers/) project.
+
+* [Introduction](#introduction)
+* [License](#license)
+* [Platform support](#platform-support)
+    * [Hardware requirements](#hardware-requirements)
+* [Quick start for developers](#quick-start-for-developers)
+* [Configuration](#configuration)
+* [Logging](#logging)
+* [Debugging](#debugging)
+* [Community](#community)
+* [Home Page](#home-page)
+
+## Introduction
+
+`kata-runtime`, henceforth known simply as "the runtime", is the
+CLI (command-line interface) part of the Kata Containers runtime component. It leverages the
+[virtcontainers](https://github.com/kata-containers/runtime/tree/master/virtcontainers)
+package to provide a high-performance standards-compliant runtime that creates
+hardware-virtualized containers.
+
+The runtime is both
+[OCI](https://github.com/opencontainers/runtime-spec)-compatible and
+[CRI-O](https://github.com/kubernetes-incubator/cri-o)-compatible, allowing it
+to work seamlessly with both Docker and Kubernetes respectively.
+
+## License
+
+The code is licensed under an Apache 2.0 license.
+
+See [the license file](LICENSE) for further details.
+
+## Platform support
+
+Kata Containers currently works on systems supporting the following
+technologies:
+
+- [Intel](https://www.intel.com)'s VT-x technology.
+- [ARM](https://www.arm.com)'s Hyp mode (virtualization extension).
+
+### Hardware requirements
+
+The runtime has a built-in command to determine if your host system is capable
+of running a Kata Container:
+
+```bash
+$ kata-runtime kata-check
+```
+
+> **Note:**
+> 
+> If you run the command above as the `root` user, further checks will be
+> performed (e.g. it will check if another incompatible hypervisor is running):
+> 
+> ```bash
+> $ sudo kata-runtime kata-check
+> ```
+
+## Quick start for developers
+
+See the
+[developer guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md).
+
+## Configuration
+
+The runtime uses a TOML format configuration file called `configuration.toml`.
+The file contains comments explaining all options.
+
+> **Note:**
+> 
+> Most users will not need to modify the default configuration.
```

Comment:
I'm not sure how true this is - maybe we can rephrase it more like:
`The defaults provide a good default configuration. You may only need to modify this file if you have specialist advanced user needs`
or similar (that still needs work...)